### ISSUE TITLE
To Readme, add to "special thanks for" docs with some links (archive.org in this case) Sample

### ISSUE THREAD

**Issue by cuvtixo** [cbmeeks/TMS9918#3]
TMS9918
Various documents for the TMS9918/TMS9928 and TMS9958 VDP's that I've collected over the years.

Useful Links
http://forum.6502.org/viewtopic.php?f=4&t=3564&start=0

https://hackaday.io/project/5789-6502-homebrew-computer/log/18434-video-output

Great example of using the TMS in a homebrew computer based off the 6809:

https://github.com/74hc595/Ultim809/tree/master/circuit

Special Thanks
Special Thanks to David Anderson (factus10) for providing these files:

1980_Arizona_Technical_Symposium_Draft.pdf
CMU-ECE-1984-010.pdf
Oliger TMS9918 Timex 1000 (SyncWare News).pdf
TI-VDP-PRG.pdf
tms9918a.txt
And thanks goes to bissonex for discovering a missing page in the Apple II Byte article (Hi-Res Graphics TMS9918 BYTE 0882.pdf). https://archive.org/details/Hi-Res_Graphics_TMS9918_BYTE_0882/page/n1/mode/2up

https://github.com/bissonex

**Comment by cbmeeks** [comment]
I'm not sure what you mean.  All of the documents I have uploaded are the documents I have collected from various sources over the years.  None came directly from Archive.org.  

The exceptions have already been credited.

**Comment by cuvtixo** [comment]
> I'm not sure what you mean. All of the documents I have uploaded are the documents I have collected from various sources over the years. None came directly from Archive.org.
> 
> The exceptions have already been credited.

Oh, sorry. that was worded badly by me. I was asking for links to archive.org IN ADDITION to links already existing. There's good things there. Why wouldn't you want to add to the collection you started? I only mentioned "special thanks for" because that's how the last section is labelled presently. Whatever you call any additions or where in the README you put them doesn't matter. I don't expect anyone expects or needs thanks for any additional links. I'm just thinking spacewise they'd fit on the bottom, especially if you don't consider them as valuable as the present "useful links".

I think this was a three o'clock in the morning idea.  I thought there's probably a ton of material on Archive.org, I just wasn't prepared in the moment to look them up myself.  I mean, is your project meant for you specifically to gather docs you found? Or is it meant to be a public resource?  I can certainly fork the project and add whatever I find at Archive.org myself.  I just thought of sharing the possibility. It doesn't look like anything you have here is from Archive.org and that's specifically what the site is for. Archiving documents. (and possibly software. there's "free" software of all kinds.

**Comment by cbmeeks** [comment]
Well, this repo started one day when I noticed people were talking about the TMS9918 in various forums.  I had already been collecting documents about it for several years.  So I thought I would share my collection and told a few people on forums.

Shortly after, a couple people sent me stuff I didn't have and I also found some links where people used the TMS9918 in various projects.  Which I included.

I searched Archive.org for "tms9918" and I only see three documents about it.  Which I already have here.  There are a few games also listed but this repo isn't about listing all games created with the TMS9918.  

If you have specific links to any TMS9918 related document that I don't have here, I will certainly be glad to add it.

Thanks.
